src/style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+
+  --imac-photo-width: 160px;
+  --imac-photo-height: 134px;
+}
+
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__photo {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+
+.product-card__imac-photo {
+  display: block;
+  width: var(--imac-photo-width);
+  height: var(--imac-photo-height);
+
+  margin-bottom: 40px;
+}
+
+.product-card__imac-about {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+
+  color: #060b35;
+}
+
+.product-card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.product-card__grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: #060b35;
+}
+
+.product-card__stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star-active.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.star--5 {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg);
+  background-size: auto;
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
```

Comment:
Look at the tips & hints for this task: there should be BEM blocks for card and stars.